eZ/Bundle/EzPublishCoreBundle/Resources/views/content_fields.html.twig

```diff
@@ -403,6 +403,8 @@
 
 {# This field accepts the following parameters:
  #   - alias (image variation name). Defaults to "original" (e.g. image originally uploaded)
+ #   - parameters.width. Allows forcing width of the image
```

Comment:
I'd have specified "in the HTML". It doesn't force the width of the image itself, but of the HTML element that shows the image.